crowdin.yml

```diff
@@ -34,91 +34,91 @@ files: [
     "translation" : "/wire-ios/Wire-iOS/Resources/Localization/%osx_code%/%original_file_name%",
   },
   {
-    "source" : "/wire-ios/Wire-iOS Share Extension/Resources/Localization/Base.lproj/Localizable.strings",
+    "source" : "/wire-ios/Wire-iOS Share Extension/Resources/Base.lproj/Localizable.strings",
     "dest" : "/App/Wire-iOS Share Extension/Localizable.strings",
     "translation" : "/wire-ios/Wire-iOS Share Extension/%osx_code%/%original_file_name%",
```

Comment:
I wonder if this line should be
```
"translation" : "/wire-ios/Wire-iOS Share Extension/Resources/%osx_code%/%original_file_name%",
```

Reply:
Added the change to this PR:
https://github.com/wireapp/wire-ios/pull/2603/files